compiler/src/iree/compiler/Tools/init_input_passes.cc

```diff
@@ -10,6 +10,7 @@
 
 #ifdef IREE_HAVE_MHLO_INPUT
 #include "iree/compiler/InputConversion/MHLO/Passes.h"
+#include "iree/compiler/InputConversion/StableHLO/Passes.h"
```

Comment:
Should we separate this out into its own input rather than reusing defines and stuff for MHLO?

Reply:
Right now it depends on the copy of stablehlo from mhlo, so I'm not sure if it makes sense to introduce a separate define. Once we make substitute the mlir-hlo-fork submodule for stablehlo, we can rename it. Are there other reasons why we may want to have 2 defines?

Reply:
> and we expect to only have one dependency (git submodule / code directory) for both, right?

Yes